Automattic/jetpack
# SSO: Add JS and CSS tooltips

Fixes #

## Proposed changes:

* Improve the tooltips by using JS and CSS instead of relying on HTML title attribute.

How it is now:

https://github.com/Automattic/jetpack/assets/2653810/0021e3bd-3284-4afa-ae0d-1f9312a0b5a3



### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync this branch into your atomic site
* Navigate to `/wp-admin/users.php`
* Make sure you have a pending invite to be able to see the pending invite exclamation mark
<img width="188" alt="image" src="https://github.com/Automattic/jetpack/assets/2653810/57bb177d-64c1-4a0a-8e86-81408957836f">

* Hove over the tooltip on the table header
 
<img width="198" alt="image" src="https://github.com/Automattic/jetpack/assets/2653810/11dbea3c-4826-46c7-94b6-35ca70791763">
